src/shorthands/padding.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import directionalProperty from '../helpers/directionalProperty'
+
+/**
+ * The padding shorthand accepts up to four values, including null to skip a value, and uses the directional-property mixin to map them to their respective directions.
+ * @example
+ * // Styles as object usage
+ * const styles = {
+ *   ...padding('12px', '24px', '36px', '48px')
+ * }
+ *
+ * // styled-components usage
+ * const div = styled.div`
+ *   ${padding('12px', '24px', '36px', '48px')}
+ * `
+ *
+ * // CSS as JS Output
+ *
+ * div {
+ *   'padding-top': '12px',
+ *   'padding-right': '24px',
+ *   'padding-bottom': '36px',
+ *   'padding-left': '48px'
+ * }
+ */
+
+function padding(...values: Array<string|null>) {
```

Comment:
If you set `null` as a type flow will think that's an `any`, which we need to avoid. Instead of saying that, do `param: ?string`. That'll tell flow that the param is null-able!